backend/app/models/analytics_model.go

```
@@ -7,49 +7,52 @@ import (
 )
 
 func init() {
+	// Only auto-migrate URLMetric, removing the old tables
 	db.GetDB().AutoMigrate(&URLAnalytics{})
```

golangci-lint (1.64.8)

Error return value of `(*gorm.io/gorm.DB).AutoMigrate` is not checked

(errcheck)